app/src/features/rules/components/RulesList/components/RulesTable/utils/rules.ts

```diff
@@ -32,6 +32,23 @@ export const getActiveRules = (records: RuleObj[]) => {
   return [...activeRecords, ...rules];
 };
 
+export const getPinnedRules = (recordsMap: Record<string, RuleObj>) => {
```

Comment:
```suggestion
export const getPinnedRules = (recordsMap: Record<RuleObj.id, RuleObj>) => {
```
Will lead to a slightly more informative intellisense